pkg/operatorsource/handler.go

```diff
@@ -2,12 +2,21 @@ package operatorsource
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
 	"github.com/operator-framework/operator-sdk/pkg/sdk"
-	"github.com/sirupsen/logrus"
 )
 
+// NewHandlerWithParams returns a new Handler
+func NewHandlerWithParams(strategy PhaseReconcilerStrategy, kubeclient Kubeclient, transitioner PhaseTransitioner) Handler {
+	return &handler{
+		strategy:     strategy,
+		kubeclient:   kubeclient,
+		transitioner: transitioner,
+	}
+}
+
 // Handler is the interface that wraps the Handle method
 //
 // Handle handles a new event associated with OperatorSource type
```

Comment:
Please move this comment above the `Handle()` function

Reply:
I think I answered this in a previous comment - this is the idiomatic way of documenting single method interface.

Reply:
So I will still see the documentation if using an editor and Sourcegraph?